### PR TITLE
Tperps 865 fetch from api

### PR DESCRIPTION
**Pull request by dospore**
### Motivation

Parse OME data from API

### Changes

- update readme and env variable
- deconstruct data from API
- fillter on new status's

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ If you are not making changes to the graph or the OME, you can run off the deplo
 ```
 NEXT_PUBLIC_TRADER_ADDRESS="0x98D801b0cB3576c048CB74e095187DF5E7025D61"
 NEXT_PUBLIC_GRAPH_URI=https://api.thegraph.com/subgraphs/name/tracer-protocol/tracer-kovan
-NEXT_PUBLIC_OME_BASE_URL=https://order.tracer.finance
+NEXT_PUBLIC_API_URL=https://order.tracer.finance
```
Comment (chrdevmar) [inline README.md]:
```suggestion
NEXT_PUBLIC_API_URL=https://api.tracer.finance
```

File: libs/types/General.ts [hunk 2]
```diff
@@ -15,3 +15,9 @@ export type Result = {
     message?: string;
     error?: string;
 };
+
+export type APIResult = {
+    status: 'error' | 'success';
```
Comment (chrdevmar) [inline libs/types/General.ts:20]: did you observe this in one of the responses? I wouldn't have thought anything should have this

**Review by chrdevmar — CHANGES_REQUESTED** [review]

File: context/TracerContext.tsx [hunk 3]
```diff
@@ -153,10 +153,10 @@ export const SelectedTracerStore: React.FC<StoreProps> = ({ tracer, children }:
             const omeOrder = orderToOMEOrder(web3, await signedMakes[0]);
             console.info('Placing OME order', omeOrder);
             const res = await createOrder(selectedTracer?.address as string, omeOrder);
-            if (res.message === 'PartialMatch' || res.message === 'FullMatch') {
+            if (res.data === 'matched_partial' || res.data === 'matched') {
```
Comment (chrdevmar) [inline context/TracerContext.tsx]:
```suggestion
            if (res.data.status === 'matched_partial' || res.data.status === 'matched') {
```

**Review by chrdevmar — CHANGES_REQUESTED** [review]

File: context/OMEContext.tsx [hunk 4]
```diff
@@ -25,9 +25,9 @@ export const parseOrders: (res: any) => Orders = (res) => {
         const sections = Object.values(orders);
         const flattenedOrders = sections.map((orders: any) =>
             orders.reduce(
-                (prev: any, order: { amount_left: number; price: number }) => ({
+                (prev: any, order: { remaining: number; price: number }) => ({
                     price: parseFloat(Web3.utils.fromWei(order.price.toString())), // price remains the same,
-                    quantity: prev.quantity + parseFloat(Web3.utils.fromWei(order.amount_left.toString())),
+                    quantity: prev.quantity + parseFloat(Web3.utils.fromWei(order.remaining.toString())),
```
Comment (chrdevmar) [inline context/OMEContext.tsx:30]: `amount_left` changing to `remaining` is unexpected, where did you see this?

File: libs/types/General.ts (hunk 2 above)
Comment (dospore) [reply libs/types/General.ts:20]: nah this was just so I could fix it for testing with the least amount of work. A lot of the error handling on the front end uses these status's

File: context/OMEContext.tsx (hunk 4 above)
Comment (dospore) [reply context/OMEContext.tsx:30]: https://api.tracer.finance/book/1113ef6fcbbece1fa19155ba8deb223f88f5c266

Comment (dospore) [reply context/OMEContext.tsx:30]: I created an order just then

Comment (dospore) [reply context/OMEContext.tsx:30]: may have to ask Jack what it 'should' return

File: context/TracerContext.tsx (hunk 3 above)
Comment (dospore) [reply context/TracerContext.tsx]: Will need to update the message as well

File: context/OMEContext.tsx (hunk 4 above)
Comment (chrdevmar) [reply context/OMEContext.tsx:30]: oh I see, its `remaining` when fetching from the OME, but its stored in the database as `amount_left`. might be good to change one of them to be consistent

Comment (dospore) [reply context/OMEContext.tsx:30]: definitely

**Review by chrdevmar — DISMISSED** [review]

**Pull request merged**